spec/support/capybara_helpers.rb

```diff
@@ -62,6 +62,12 @@ def unfocus
     find("body").click
   end
 
+  def fill_in_datetime(field, with:)
+    element = find_field(field)
+    element.click
+    element.execute_script("this.value = arguments[0]; this.dispatchEvent(new Event('blur', {bubbles: true}));", with)
+  end
+
```

Comment:
Added a `fill_in_datetime` helper for testing datetime-local inputs in React components. React's DateInput component only processes the input value in its onBlur handler. Capybara's standard fill_in doesn't trigger blur events, causing the value to not be saved. This helper explicitly triggers the blur event after setting the value, ensuring React properly processes and saves the datetime input.